pkg/autoscaler/statforwarder/forwarder_test.go

```diff
@@ -166,6 +166,14 @@ func TestForwarderReconcile(t *testing.T) {
 	}); err != nil {
 		t.Fatal("Timeout to get the Endpoints:", lastErr)
 	}
+
+	// Wait for websocket connection to created before terminating the test to avoid logging after test.
+	if err := wait.PollImmediate(10*time.Millisecond, 2*time.Second, func() (bool, error) {
+		p1 := f1.getProcessor(bucket1)
+		return p1 != nil && p1.conn != nil, nil
```

Comment:
isn't this sync now?

Reply:
See https://github.com/knative/serving/pull/9472#discussion_r494600407